project_resstock_national_upgrades.yml

```diff
@@ -1,13 +1,32 @@
-schema_version: 0.2
+schema_version: '0.3'
 buildstock_directory: ../resstock  # Relative to this file or absolute
 project_directory: project_resstock_heterogeneity  # Relative to buildstock_directory
 output_directory: /scratch/nmerket/heterogeneity4
-#weather_files_url: https://s3.amazonaws.com/epwweatherfiles/project_resstock_national.zip
-weather_files_path: /home/nmerket/buildstock/assets/project_resstock_national_weather.zip  # Relative to this file or absolute path to zipped weather files
+# weather_files_url: https://data.nrel.gov/system/files/128/ResStock_TMY3.zip
```

Comment:
Update url to https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip to be compatible with resstock 2.4.0.